changelog.txt

```diff
@@ -2,6 +2,7 @@
  * Added Realm.copyFromRealm() for creating detached copies of Realm objects.
  * Added RealmObjectSchema.getFieldType()
  * Added unitTestExample to showcase unit and instrumentation tests. Examples include jUnit3, jUnit4, Espresso, Robolectric, and MPowermock usage with Realm (#1440).
+ * Added RealmList#contains() method that utilizes core find() method to help with contains() performance (#897).
```

Comment:
Contains have always been present. Perhaps just "Improved performance of RealmList.contains (#897)"